Sources/Basics/AsyncFileSystem/AsyncFS.swift

```diff
@@ -11,11 +11,10 @@
 //===----------------------------------------------------------------------===//
 
 import protocol _Concurrency.Actor
-import protocol Crypto.HashFunction
-import struct SystemPackage.Errno
-import struct SystemPackage.FilePath
+@preconcurrency import struct SystemPackage.Errno
+@preconcurrency import struct SystemPackage.FilePath
 
-package protocol AsyncFileSystem: Actor {
+package protocol AsyncFS: Actor {
```

Comment:
Renamed for consistency, as we also had to rename `VirtualFileSystem` to avoid collisions with its sync counterpart in the `Basics` module.